### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Update Target pop up_Screenreader : VoiceOver reads the background content when popup is opened. 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6871]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #MAS1.3.1; #MAS1.3.2; #VoiceOver;

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
Additional Details :
Tools: MAC 10.14.6, Voice Over
Mas reference: MAS1.3.1, MAS1.3.2

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target. In target select any database.
10. Navigate till Ok button & hit enter.
11. Navigate till Compare menu from Schema compare Menu bar & hit enter.
12. Navigate till Apply button from Schema Compare Menu bar & hit enter.
13. Update Target pop up will appear
14. Start Voice over and use Control+Option+Arrow keys and check whether it reads the content or not.

**Actual:**
Using Control+Option+Arrow keys Voiceover is not reading the background content of the popup.

**Expected:**
Using Control+Option+Arrow keys Voiceover focus should move to inside the popup and it should not read the background content.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screenreader user will not be able to read the whole content of the popup.

**MAS Reference:**
MAS1.3.1-https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

MAS1.3.2- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

**Attachment for Reference:**
![6871_A11y_AzureDataStudio(Schema,Dacpac,Backup)_Update Target pop up_Screenreader](https://user-images.githubusercontent.com/24580570/63510283-cefc7e00-c4fb-11e9-8f82-9c870e90d429.png)

**Comment by udeeshagautam** [comment]
Is this voiceOver specific? Keeping on hold with other reader specific issues

**Comment by fsteffi** [comment]
@udeeshagautam Specific to Voice over

**Comment by udeeshagautam** [comment]
This behaves same as VScode

**Comment by udeeshagautam** [comment]
@fsteffi This is same behavior as VS code. Any modal notification dialog in vscode is behaves same with voice over.  Opened an issue on VSCode : https://github.com/microsoft/vscode/issues/82727. 

@hannahqin can you please help with the external review process. Thanks!

**Comment by hannahqin** [comment]
Marking this for Exclusion. As stated above, this is a known issue with Electron and is being tracked here: electron/electron#14234.

**Importance of Experience:** minimal
Warning popups may appear in scenarios where we want to guarantee that the user's previous action was intended. In this case, it was to ensure they want to 'Apply' changes to a Target database, which would overwrite any existing data. These warning scenarios occur very infrequently.

**Quality of Experience:** annoying
Since users are unable to access the warning notification, they will not be able to follow through with applying these changes to the Target database. They can still complete the scenario by instead selecting "Generate Script" and then running the script in order to apply the changes.

**AAT Conformance Doc:** https://cela-ra-aat-prod.azurewebsites.net/#/workflow/ee5c9007-47f4-4fcb-a4d7-a998a617edde?tab=default

**@fsteffi, please work with a SME to review.**

**Comment by kenvanhyning** [comment]
Closing as upstream

**Comment by Alexandria-R** [comment]
Approved to close as non blocking.
